blogvault/wpcom-migration
# Add the static main page

## Proposed Changes
* Add the main static page.
* Propose a scaffold for the static pages.
* Add some temporary code for loading the static pages in the context of WordPress.

## Screenshots

Desktop:

<img width="1496" alt="image" src="https://github.com/user-attachments/assets/4a56194a-abaf-4038-a63e-98d76f3b5ff4">

iPhone 12 (horizontal):

<img width="630" alt="image" src="https://github.com/user-attachments/assets/97ba9073-906c-487d-bd00-20a00dc23f15">

iPhone 12 (vertical):

<img width="289" alt="image" src="https://github.com/user-attachments/assets/00de5085-337a-41b3-a8c9-84072121be4f">



## Testing Instructions

1. [Download](https://download-directory.github.io/?url=https%3A%2F%2Fgithub.com%2Fblogvault%2Fwpcom-migration%2Ftree%2Fadd%2Fstatic-main-page%2Fsrc) and install the plugin. You could use [jurassic.ninja](https://jurassic.ninja/) for a quick environment setup.
2. Go to `/wp-admin/admin.php?page=automattic&static=main`.
3. Compare the page to the [design](https://www.figma.com/design/c9jLiGBEqrxgEpHc8ccVFr/WP.com-migration-plugin-screen-designs?node-id=163-411&t=E27OyiNeJR3nqdoN-0). Having small differences should be fine, IMO.
4. Make sure it looks good on mobile and the different browsers.
